frame/executive/src/lib.rs

```diff
@@ -895,13 +896,15 @@ mod tests {
 
 	#[test]
 	fn validate_unsigned() {
-		let xt = TestXt::new(Call::Balances(BalancesCall::set_balance(33, 69, 69)), None);
+		let xt = TestXt::new(Call::Custom(custom::Call::allowed_unsigned()), None);
 		let mut t = new_test_ext(1);
 
+		let mut default_with_prio_3 = ValidTransaction::default();
+		default_with_prio_3.priority = 3;
```

Comment:
Why 3? :D Is that coming from `CheckWeight`?

Reply:
yes come from it, though I didn't read the implementation carefully.